### DISCUSSION TITLE
How to use the subpath-based proxy

### DISCUSSION THREAD

**Discussion by thincal** [coder/code-server#4393] [General]
I have one proxy server managing the vscode server instance, and providing the access to them. For the end user he can find out the vscode instance id and access by `https://myproxyserver.com/vss/{vsId}`, but how is vscode server reached with the subpath-based proxy? just like this`192.168.0.23:8080/absproxy/8080/vs1` ?

**Comment by jsjoeio** [comment]
Hmm...I'm not sure I fully understand. Maybe @code-asher has an idea?

**Comment by code-asher** [comment]
So if you have a proxy set up that can allow access to code-server at
`[domain]/vss/{id}` then the subpath port proxy can be used by going to
`[domain]/vss/{id}/proxy/{port}` or `[domain]/vss/{id}/absproxy/{port}`.
